mkdocs.yaml

```diff
@@ -1,8 +1,9 @@
 # ---------------------- PROJECT SPECIFIC ---------------------------
 
 site_name: DataJoint Documentation
-repo_url: https://github.com/datajoint-company/datajoint-docs
+site_url: http://localhost/docs
```

Comment:
```suggestion
site_url: https://www.datajoint.com/docs
```

Reply:
Perhaps this suggestion is not required based on how we deploy the site.

Reply:
Thanks @kabilar but this was intentional. Using `localhost` should protect us from future updates to the domain without requiring cascading PR's (though subdomain is a different story). I don't expect it to affect the deployment.

Reply:
Thank you